engine-standalone-storage/src/engine_state.rs

```diff
@@ -102,7 +102,7 @@ impl<'db, 'input: 'db, 'output: 'db> IO for EngineStateAccess<'db, 'input, 'outp
         let value = iter.next().and_then(|maybe_elem| {
             maybe_elem
                 .ok()
-                .map(|(_, value)| DiffValue::try_from_bytes(&value).unwrap())
+                .map(|(_, value)| DiffValue::try_from_bytes(&value).expect("value should be valid"))
```

Comment:
```suggestion
                .map(|(_, value)| DiffValue::try_from_bytes(&value).expect("diff value is invalid"))
```

Reply:
Fixed in https://github.com/aurora-is-near/aurora-engine/pull/916/commits/28be3e2a84ba8797f1f8838f4e45f1e98e16a1f9